AppController/djinn.rb

```diff
@@ -487,7 +487,7 @@ class Djinn
 
   # A String that is returned to callers of set_property that provide an invalid
   # instance variable name to set.
-  KEY_NOT_FOUND = "No property exists with the given name."
+  KEY_NOT_FOUND = "Invalid property name, or property value."
```

Comment:
@jeanleonov just to make sure I understood: are you saying that the new message is more clear? or that the is more confusing :) And if you are suggesting the latter, what would you recommend?